mod/ckeditor/views.php

```diff
@@ -2,7 +2,6 @@
 
 return [
 	'default' => [
-		'ckeditor.js' => __DIR__ . '/vendors/ckeditor/ckeditor.js',
```

Comment:
I have a file that replaces it: https://github.com/Elgg/Elgg/pull/10753/files#diff-e14678e40ead87449b33d84e090faddb.